src/DIPS.Xamarin.UI/Controls/Tag/Tag.xaml.cs

```diff
@@ -0,0 +1,310 @@
+﻿using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace DIPS.Xamarin.UI.Controls.Tag
+{
+    /// <summary>
+    ///     A <c>Frame</c> wrapping a <c>Label</c> to allow setting border radius for the <c>Label</c>.
+    /// </summary>
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class Tag
+    {
+        /// <inheritdoc />
+        public Tag()
+        {
+            InitializeComponent();
+        }
+
+        #region Label Properties
+
+        /// <summary>
+        ///     Bindable property for <see cref="Text" />
+        /// </summary>
+        public static readonly BindableProperty TextProperty =
+            BindableProperty.Create(nameof(Text), typeof(string), typeof(Tag), default(string));
+
+        /// <summary>
+        ///     Bindable property for <see cref="TextColor" />
+        /// </summary>
+        public static readonly BindableProperty TextColorProperty =
+            BindableProperty.Create(nameof(TextColor), typeof(Color), typeof(Tag), Color.Default);
+
+        /// <summary>
+        ///     Bindable property for <see cref="FontFamily" />
+        /// </summary>
+        public static readonly BindableProperty FontFamilyProperty =
+            BindableProperty.Create(nameof(FontFamily), typeof(string), typeof(Tag), default(string));
+
+        /// <summary>
+        ///     Bindable property for <see cref="FontAttributes" />
+        /// </summary>
+        public static readonly BindableProperty FontAttributesProperty = BindableProperty.Create(nameof(FontAttributes),
+            typeof(FontAttributes), typeof(Tag), FontAttributes.None);
+
+        /// <summary>
+        ///     Bindable property for <see cref="FontSize" />
+        /// </summary>
+        public static readonly BindableProperty FontSizeProperty =
+            BindableProperty.Create(nameof(FontSize), typeof(double), typeof(Tag), -1d);
```

Comment:
Use the same default value as `Label.FontSizeProperty.DefaultValue`. That way the Tag will have equal size as a Label when newing it up.

Reply:
`Label.FontSizeProperty.DefaultValue` is also `-1d`

Reply:
however it'll be nicer if I use it via that property 🙂👍

Reply:
will follow using the property for all other properties too

Reply:
I have tested the framework default constants and they seemed to work good. Changed them all